cypress/e2e/additems.spec.cy.js

```diff
@@ -0,0 +1,49 @@
+const { baseUrl } = require('../variables/makeup_vars.js');
+
+describe('Добавление товаров в корзину', () => {
+  beforeEach(() => {
+    // Открываем URL маркетплейса
+    cy.visit(baseUrl);
+  });
+
+  it('Проверка открытия маркетплейса', () => {
+    // Проверяем, что URL соответствует ожидаемому
+    cy.url().should('eq', 'https://makeup.com.ua/ua/');
```

Comment:
Better to add "https://makeup.com.ua" as a BaseUrl in your config file. And verification will look like this:
` cy.url().should('eq', '/ua/');`

Reply:
Fixed. Added a function to the command file and applied to all tests for verification